mass_mailing_unsubscribe_af/i18n/sv_SE.po

```diff
@@ -154,12 +154,12 @@ msgstr "Importera"
 #. module: mass_mailing_unsubscribe_af
 #: model:ir.model,name:mass_mailing_unsubscribe_af.model_import_deregistration_file
 msgid "Import Deregistration File"
-msgstr ""
+msgstr "Importera avregistreringsfil"
```

Comment:
If we are following the pattern from the english then it should be an A and not a in avregisteringsfil